changelog.md

```diff
@@ -1,9 +1,12 @@
 # Change Log
 
-## `head`
+## 'head'
```

Comment:
You should use backtick quotes(`) for these headers.

Reply:
done